backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/DefaultAndroidInput.java

```diff
@@ -992,10 +991,6 @@ public void onDreamingStarted () {
 	@Override
 	public void onDreamingStopped () {
 		unregisterSensorListeners();
-		// erase pointer ids. this sucks donkeyballs...
-		Arrays.fill(realId, -1);
```

Comment:
Do we know why only `realId` and `touched` were filled with -1 and false, respectively? No other arrays were, which seems a little odd.

Reply:
Not exactly, some of these hacks occured quite a long time ago (https://github.com/obigu/libgdx/commit/82681a44155e1db36c51714a7ce3e4f1f004385c) but my guess is that issues appeared due to no proper handling of touch cancel events and this kind of fixed it.

Reply:
It looks like the `realId` erasure is only for the Motorola Jordan? Yeah this should be fine to get rid of, I doubt any phones available in 2011 are still buying apps from the Play Store today.